cmake/find_ros1_package.cmake

```diff
@@ -20,6 +20,14 @@ macro(find_ros1_package name)
       "${ARG_UNPARSED_ARGUMENTS}")
   endif()
 
+  find_package(PkgConfig)
+  if(NOT PKG_CONFIG_FOUND)
+    message(WARNING "Failed to find PkgConfig, skipping...")
+    # call ament_package() to avoid ament_tools treating this
+    # as a plain CMake pkg
+    ament_package()
+    return()
```

Comment:
This CMake extra file can't call `ament_package()`. Instead the `find_package(PkgConfig)` call should be `REQUIRED`.